upp-factset-provisioner/sh/provision.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo $VAULT_PASS > /ansible/vault.pass
+
+cd /ansible
+
+ansible-playbook --vault-password-file=vault.pass provision.yml --extra-vars "\
```

Comment:
Similarly, we should probably be passing through the `tag_environment` here

Reply:
same as above